tests/test_benchmark_KPI.py

```diff
@@ -172,59 +172,103 @@ def test_benchmark_Economic_KPI_C2_E2(self, margs):
+        #     os.path.join(TEST_OUTPUT_PATH, use_case, expected_value_file), sep=","
+        # )
 
-        # Store to csv to enable manual check, eg. of LCOE_A. Only previously empty rows have been changed.
-        expected_values.drop("flow").to_csv(
-            os.path.join(TEST_OUTPUT_PATH, use_case, expected_value_file), sep=","
-        )
+        # Compare asset costs calculated in C2 and E2 with benchmark data from csv file
+        for asset in expected_values.index:
 
-        # Check if asset costs were correctly calculated in C2 and E2
-        for asset in expected_values.columns:
-            # determine asset dictionary (special for storages)
+            asset_group = expected_values.loc[asset, "group"]
+
+            # determine asset dictionnary (special for storages)
             if asset in [INPUT_POWER, OUTPUT_POWER, STORAGE_CAPACITY]:
-                asset_data = data[expected_values[asset]["group"]]["storage_01"][asset]
+                asset_data = data[asset_group]["storage_01"][asset]
             else:
-                asset_data = data[expected_values[asset]["group"]][asset]
+                asset_data = data[asset_group][asset]
             # assertion
             for key in KEYS_TO_BE_EVALUATED:
-                assert expected_values[asset][key] == pytest.approx(
+                assert expected_values.loc[asset, key] == pytest.approx(
                     asset_data[key][VALUE], rel=1e-3
                 ), f"Parameter {key} of asset {asset} is not of expected value."
 
+        demand = pd.read_csv(
+            os.path.join(TEST_INPUT_PATH, use_case, TIME_SERIES, "demand.csv"), sep=",",
+        )
+        aggregated_demand = demand.sum()[0]
+
+        # Compute the aggregated annuity and costs (NPC)
+        aggregated_annuity = 0
+        aggregated_costs = 0
+
+        for asset_group in (ENERGY_CONSUMPTION, ENERGY_PRODUCTION, ENERGY_STORAGE):
+            for asset in data[asset_group]:
+                # for storage we look at the annuity of the in and out flows and storage capacity
+                if asset_group == ENERGY_STORAGE:
+                    for storage_type in [INPUT_POWER, OUTPUT_POWER, STORAGE_CAPACITY]:
+                        asset_data = data[asset_group][asset][storage_type]
+                        aggregated_annuity += asset_data[ANNUITY_TOTAL][VALUE]
+                        aggregated_costs += asset_data[COST_TOTAL][VALUE]
+                else:
+                    asset_data = data[asset_group][asset]
+                    aggregated_annuity += asset_data[ANNUITY_TOTAL][VALUE]
+                    aggregated_costs += asset_data[COST_TOTAL][VALUE]
+
+        # Compute the lcoe for this simple case (single demand)
+        lcoe = aggregated_annuity / aggregated_demand
+        mvs_lcoe = data[KPI][KPI_SCALARS_DICT][
+            "Levelized costs of electricity equivalent"
+        ]
+        assert lcoe == pytest.approx(
+            mvs_lcoe, rel=1e-2
```

Comment:
That rel is too large - lcoe is between 0.04 and 1.2, and a deviation of 0.01 is too much. Maximum acceptable one is 0.001, if possible even less

Reply:
Ok I will change that